### ISSUE TITLE
The "calendar" modal lacks in accessibility

### ISSUE THREAD

**Issue by Arnei** [elan-ev/opencast-editor#267]
The time and date pickers in the metadata tab are not fully accessible. The main issues are very weird screen reader output and some parts of the UI (e.g. time selector) cannot be controlled by keyboard. -Wien

**Comment by Arnei** [comment]
This is an issue with the date and time picker package by material-ui. Unfortunately, programming such a tool is quite cumbersome, so I would like to keep relying on external packages. Even more unfortunately, in my research I was unable to find a more accessible tool.

Therefore, this is very likely to not get fixed (at least not by this project). However, all dates and times should also be fully editable by keyboard, without accessing the picker modals at all. Hopefully this allows users who cannot use the picker modal to still use the metadata editor

**Comment by ziegenberg** [comment]
Which external package exactly is used? Can you provide a link? I'd like to bring this issue upstream.

**Comment by Arnei** [comment]
The exact package is `mui-rff` https://github.com/lookfirst/mui-rff.
